app/_locales/en/messages.json

```
@@ -278,7 +278,10 @@
     "description": "$1 is Learn more link"
   },
   "addNewAccount": {
-    "message": "Add a new account"
+    "message": "Add a new Ethereum account"
+  },
+  "addNewBitcoinAccount": {
+    "message": "Add a new Bitcoin account"
```

According to the copy it should have a `(Beta)` suffix.

```suggestion
    "message": "Add a new Bitcoin account (Beta)"
```